tests/helics/application_api/TranslatorTests.cpp

```diff
@@ -449,25 +449,25 @@ TEST_F(translator, translator_connections9_nosan)
     FullDisconnect();
 }
 
-TEST_F(translator, translator_connections10)
+TEST_F(TranslatorFixture, translator_connections10)
 {
     auto broker = AddBroker("test", 1);
 
     AddFederates<helics::CombinationFederate>("test", 1, broker, helics::timeZero, "A");
 
     auto cFed1 = GetFederateAs<helics::CombinationFederate>(0);
 
-    auto& e1 = cFed1->registerGlobalTargetedEndpoint("e1", "any");
-    auto& i1 = cFed1->registerGlobalInput<double>("i1");
-    auto& p1 = cFed1->registerGlobalPublication<double>("p1");
-    p1.setOption(HELICS_HANDLE_OPTION_CONNECTION_REQUIRED);
-    e1.setOption(HELICS_HANDLE_OPTION_CONNECTION_REQUIRED);
-    i1.setOption(HELICS_HANDLE_OPTION_CONNECTION_REQUIRED);
+    auto& end1 = cFed1->registerGlobalTargetedEndpoint("e1", "any");
+    auto& input1 = cFed1->registerGlobalInput<double>("i1");
+    auto& pub1 = cFed1->registerGlobalPublication<double>("p1");
+    pub1.setOption(HELICS_HANDLE_OPTION_CONNECTION_REQUIRED);
+    end1.setOption(HELICS_HANDLE_OPTION_CONNECTION_REQUIRED);
+    input1.setOption(HELICS_HANDLE_OPTION_CONNECTION_REQUIRED);
 
-    e1.addSourceTarget("t1");
-    p1.addDestinationTarget("t1");
-    i1.addSourceTarget("t1");
-    e1.addDestinationTarget("t1");
+    end1.addSourceTarget("t1");
+    pub1.addDestinationTarget("t1");
+    input1.addSourceTarget("t1");
+    end1.addDestinationTarget("t1");
```

Comment:
If we want to be consistent with the other tests, it looks like `end1` should be renamed to `endpoint1`.